sdk/test-utils/recorder-new/Roadmap.md

```diff
@@ -0,0 +1,85 @@
+# Azure @azure-tools/test-recorder-new library for JavaScript
+
+`@azure-tools/test-recorder-new` leverages the [test-proxy](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy) tool built by ENG SYS.
+
+[#15829](https://github.com/Azure/azure-sdk-for-js/issues/15829) tracks the work being done to make the `@azure-tools/test-recorder-new` tool complete.
```

Comment:
Is `test-recorder-new` the final name? I think we should schedule an arch board review of the recorder 🤔

Reply:
No, `new` is just for current distinction.
Once the development is done, this will be moved into the existing recorder package.

https://www.npmjs.com/package/@azure-tools/test-recorder
`@azure-tools/test-recorder`

Reply:
> I think we should schedule an arch board review of the recorder 🤔 I’m sure they will have feedback.

API surface wise, it is too small.
- creating a client
- plugging it in the client options or adding it to the pipeline as a policy
- starting the recorder
- stopping the recorder

Reply:
I’ll resolve this since we spoke to a greater length over teams.